packages/validation/docs/valibot-api-index.md

```
@@ -1,287 +1,421 @@
+
 - [void](https://valibot.dev/api/void/): Creates a void schema.
+
 ```
 const Schema = v.void<TMessage>(message);
 ```
 
 ## Methods
 
 - [assert](https://valibot.dev/api/assert/): Checks if the input matches the scheme.
+
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)